store/orm/orm.go

```diff
@@ -66,10 +67,37 @@ func NewORM(uri string, timeout time.Duration) (*ORM, error) {
 	if err != nil {
 		return nil, fmt.Errorf("unable to init DB: %+v", err)
 	}
-	return &ORM{
+
+	orm := &ORM{
 		DB:              db,
 		lockingStrategy: lockingStrategy,
-	}, nil
+	}
+
+	if linkstatsURL != nil {
+		db.Callback().Create().After("gorm:update").Register("sync:run_after_create", createSyncEvents)
+	}
+
+	return orm, nil
+}
+
+func createSyncEvents(scope *gorm.Scope) {
+	if scope.HasError() {
+		return
+	}
+
+	if scope.TableName() == "job_runs" {
+		bodyBytes, err := json.Marshal(scope.Value)
+		if err != nil {
+			scope.Err(err)
+			return
+		}
+
+		event := models.SyncEvent{
+			Body: string(bodyBytes),
```

Comment:
Am I correct in assuming this wouldn't include updates to task runs or any details on associations? Seems fine as a first step, but we would need to expand this eventually. You wouldn't happen to have an example of the body string that's saved would you?

Reply:
`{"id":"d19e1df47ecb40fa85e7f29b4c25cd6e","jobId":"b7dbc97018ce4652b79f3f17e20fce00","result":{"jobRunId":"","taskRunId":"","data":{},"status":"","error":null},"status":"in_progress","taskRuns":[{"id":"c4067457f59b4f3d88dda39c80178995","result":{"jobRunId":"d19e1df47ecb40fa85e7f29b4c25cd6e","taskRunId":"c4067457f59b4f3d88dda39c80178995","data":{},"status":"","error":null},"status":"","task":{"ID":1,"CreatedAt":"2019-03-25T12:33:34.955039-07:00","UpdatedAt":"2019-03-25T12:33:34.955039-07:00","DeletedAt":null,"type":"noop","confirmations":0,"params":{}},"minimumConfirmations":0}],"createdAt":"2019-03-25T12:33:34.956255-07:00","completedAt":null,"updatedAt":"2019-03-25T12:33:34.956255-07:00","initiator":{"id":1,"jobSpecId":"b7dbc97018ce4652b79f3f17e20fce00","type":"runat","CreatedAt":"2019-03-25T12:33:34.954949-07:00","params":{"time":"2018-01-08T18:12:01.103Z","ran":true,"address":"0x0000000000000000000000000000000000000000"}},"creationHeight":null,"observedHeight":null,"overrides":{"jobRunId":"","taskRunId":"","data":{},"status":"","error":null}}`

Reply:
Has `taskRuns :[...]` 👍 

Will probably need to scrub that of private info in a future story.